packages/schemas/resources/views/schema.blade.php

```diff
@@ -36,6 +36,10 @@ function (Component | Action | ActionGroup $component) use (&$hasVisibleComponen
 @endphp
 
 @if ((! $isDirectlyHidden()) && $hasVisibleComponents)
+    @if ($isRoot)
+        {!! '<div class="fi-grid-ctn">' !!}
```

Comment:
Out of curiosity, what's the diff between these two?

```blade
@if($isRoot)
    <div class="fi-grid-ctn">
@endif
```

Reply:
The Prettier formatter doesn't like that 😁

Reply:
LOL 🤦🏼‍♂️